PhysicsTools/PatAlgos/python/slimming/MicroEventContent_cff.py

```diff
@@ -78,7 +78,9 @@
         # Lumi
         'keep LumiScalerss_scalersRawToDigi_*_*',
         # CTPPS
-        'keep CTPPSLocalTrackLites_ctppsLocalTrackLiteProducer_*_*'
+        'keep CTPPSLocalTrackLites_ctppsLocalTrackLiteProducer_*_*',
+	# displacedStandAlone muon collection for EXO
+	'keep recoTracks_displacedStandAloneMuons_*_*',
```

Comment:
this will actually pick up ```recoTracks_displacedStandAloneMuons__``` and ```recoTracks_displacedStandAloneMuons_UpdatedAtVtx_``` .
The latter is not saved in production/reco.
Based on your slides https://indico.cern.ch/event/731976/contributions/3022385/attachments/1657880/2654757/180528_zhud_V2.pdf I think that this should be ```'keep recoTracks_displacedStandAloneMuons__*'```

Reply:
you're right @slava.
@vinzenzstampf can you commit this small change?